cfme/configure/configuration/region_settings.py

```diff
@@ -428,3 +428,72 @@ class MapTagsEdit(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.table.row(tag_category=self.obj.category).click()
+
+
+# ====================C and U===================================
+
+class CANDUCollectionView(RegionView):
+    """C and U View"""
+    all_clusters_cb = BootstrapSwitch(id='all_clusters')
+    all_datastores_cb = BootstrapSwitch(id='all_storages')
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.canducollection.is_active() and
+            self.all_clusters_cb.is_displayed
+        )
+
+
+class CANDUCollection(Navigatable):
+    """ Class represents a C and U in CFME UI """
+    def __init__(self, appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
+
+    def _enable_disable(self, enable=True, reset=False):
+        """ Enable/Disable C and U
+
+            Args:
+                enable: Switches states, 'True'- enable
+                reset: Reset changes, default is 'False' - changes will not be reset
+        """
+        view = navigate_to(self, 'Details')
+        view.fill({
+            'all_clusters_cb': enable,
+            'all_datastores_cb': enable
+        })
+        if reset:
```

Comment:
no, I need to change the values as reset button is inactive without any changes done